src/main/java/uk/gov/hmcts/juror/api/juror/domain/JurorResponseQueries.java

```diff
@@ -144,8 +144,7 @@ public static BooleanExpression byAssignedUrgent(User staffMember) {
      * @return all responses assigned to staff
      */
     public static BooleanExpression byAssignedAll(User staffMember) {
-        return jurorResponse.staff.isNotNull()
-            .and(notClosed())
-            .and(assignedTo(staffMember));
+        return jurorResponse.staff.eq(staffMember)
```

Comment:
Would be better to use the assignedTo method (So if we change this logic one point of change)

Reply:
Updated 😄